interface/src/app/analyze/analysis/analysisDetail.tpl.html

```diff
@@ -73,10 +73,62 @@
         <td>{{sb.sampleData[id].name}}</td>
         <td ng-repeat="at in analysis.exampleCols">{{
           sb.sampleData[id].annotations[at.typename]
-          }}</td>
+        }}</td>
       </tr>
     </tbody>
   </table>
+  <div ng-if="sb.samples.length">
+    <h4>Samples Missing Activity</h4>
+    <p>The samples below are missing activity data, either because raw data
```

Comment:
:+1:  I found this information to the user and the way it was laid out really helpful!